package.json

```diff
@@ -35,7 +35,7 @@
     "ios_saf 12.2-12.5",
     "kaios 2.5",
     "op_mini all",
-    "op_mob 64",
+    "op_mob 73",
```

Comment:
Can we pin browserslist/caniuselite first and then decide later what to do here? It's technically a breaking change although it might not be one in practice. But these changes should not be mixed in with other changes.

Reply:
FYI @MatanBobi I've reverted your change to this PR so this can be resolved separately.

This will mean this PR is blocked until resolved as the CI (and local install / tests) will fail until resolved.

Reply:
Raised https://github.com/testing-library/dom-testing-library/pull/1243 as suggested